css/style.css

```diff
@@ -1,4 +1,71 @@
-h1{
-    padding: absolute;
-    left: 50%;
+#header-img {
+  width: 300px;
+}
+#form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  display: flex;
+  list-style-type: none;
+  width: 300px;
+  justify-content: space-around;
+}
+a {
+  text-decoration: none;
+}
+main {
```

Comment:
Add an empty line between selectors for better code readability. Since you do not have any linters to check your code for such errors, you will have to manually check for them.